x-pack/plugins/observability_solution/observability_shared/public/hooks/use_breadcrumbs.ts

```diff
@@ -97,7 +97,7 @@ export const useBreadcrumbs = (
   const setBreadcrumbs = useMemo(() => {
     if (!serverless?.setBreadcrumbs) {
       return (breadcrumbs: ChromeBreadcrumb[]) =>
-        chromeSetBreadcrumbs(
+        chromeSetBreadcrumbs?.(
```

Comment:
Added this is because without this the usage of the function doesn't match what we are actually testing, see https://github.com/elastic/kibana/pull/196785/files#diff-42926237891e15e7b4f739835e1f1840aac8c7206d36b0b6680b221949be4ebeL62-L65 prior the PR that's linked, `setBreadCrumbs` could have been a null value, could have accommodations made for it when using it, but now it's always defined but at the same the invoked function still could be undefined.